src/aslm/model/devices/daq/daq_base.py

```
@@ -73,10 +73,10 @@ def __init__(self, configuration):
 
         # Initialize Variables
         #: float: Sample rate of the DAQ
-        self.sample_rate = self.daq_parameters["sample_rate"]
+        self.sample_rate = self.daq_parameters.get("sample_rate", 100000)
 
         #: float: Sweep time of the DAQ
-        self.sweep_time = self.daq_parameters["sweep_time"]
+        self.sweep_time = self.daq_parameters.get("sweep_time", 0.2)
```

I think the only place this gets used is in `daq_ni.create_camera_task()`. Every other place re-defines `sweep_time` before calling it. If we get into a situation where this is not re-defined prior to calling and the exposure time is longer than the sweep time, this may create issues. Replacing this with `self.sweep_time = None` would at least alert us to this issue, should it arise. Leaving this with a default of `0.2` would possibly create silent failures, with the microscope behaving improperly with no warning.